internal/host/transition.go

```diff
@@ -543,6 +547,10 @@ func getFailedValidations(params *TransitionArgsRefreshHost) []string {
 	return failedValidations
 }
 
+func hostIsResponsive(host *models.Host) bool {
+	return host.CheckedInAt.String() == "" || time.Since(time.Time(host.CheckedInAt)) <= 3*time.Minute
```

Comment:
you have a validation for this. `IsConnected` can't you use it? 
if you can't then you should define a const because the same value is used for `isConnected` validation